fastdds_python/test/types/CMakeLists.txt

```diff
@@ -74,6 +74,8 @@ include_directories(
     ${PROJECT_SOURCE_DIR}
     )
 
+set_property(SOURCE ${PROJECT_NAME}.i PROPERTY OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/")
```

Comment:
This property is being set but not used later on.

Reply:
it used by cmake internally